troubleshoot/kibana/capturing-diagnostics.md

```diff
@@ -1,5 +1,7 @@
 ---
 navigation_title: "Diagnostics"
+applies_to:
+  stack: all
```

Comment:
This is more related to the deployment type than to the stack itself. I would put:
```suggestion
deployment:
  ess: all
  ece: all
  self: all
  eck: all
```